corehq/apps/app_manager/templates/app_manager/partials/case_list_lookup.html

```diff
@@ -122,9 +123,7 @@
                         {% trans "Results display text" %}
                     </label>
                     <div class="col-sm-4">
-                        <input class="form-control" type="text"
-                               data-bind="value: lookup_field_header"
-                               placeholder="{% trans 'Accuracy' %}" />
+                        {% input_trans detail.short.lookup_field_header langs 'lookup_field_header' data_bind='value: lookup_field_header' %}
```

Comment:
OK, I'm going to stash this commit and drop it from the PR because it doesn't achieve what I was hoping for. I'll take a look at the JS version of the "input + lang button" widget later today. 

I'm happy for this to be merged as-is; I think we're 99% there. If we get the nicer widget working we can add it in a separate PR, if this PR has already been merged.